Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -36,12 +50,13 @@ class BackForwardListViewController: UIViewController, UITableViewDataSource, UI
     }()
     
     var tabManager: TabManager!
-    var bvc: BrowserViewController!
+    weak var bvc: BrowserViewController!
```

Comment:
Since this is `weak` now, there is no guarentee that this will always have a value so we should  `!` ➡️ `?`